pages/css/inheritance.md

```diff
@@ -18,8 +25,23 @@ sidenav: css
   }
   ```
 
+You can then use the mixin in other classes, without having to rewrite the rules:
+  ```scss
+  .media-block {
+    @include clearfix;
+    ...
+  }
+  ```
+
+Mixins can also take parameters.
+
+For further guidance, see the [Sass documentation](https://sass-lang.com/guide).
+
+### When to use mixins
+
+- Use mixins for groups of properties that appear together intentionally and are used multiple times.
 - Use mixins for components to change size.
```

Comment:
I actually don't know what this means. Anyone care to clarify?

Reply:
I don't either. I think this is trying to quickly explain why inheritance is great, but that really just comes down to "use mixins when you're repeating yourself a lot" and I don't think a lengthy explanation is probably necessary here...  I think most people who are decent with CSS are already _really_ aware of repeating themselves and inherited/cascading effects in general. So I like your addition, but would probably strike the "change size" bit. I think the bigger issue is the difference between include and extend, which is tricky to understand and has huge implications to the final CSS